### ISSUE TITLE
[Suggestion] Handheld forward direction indicator

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4532]
Building a handheld gun in the correct orientation is vital, but it's easy to overlook the small visual cue of the handle trigger and just start building in any direction. Today a returning player had this issue which inspired this suggestion post.

Like with the docking connector recently, the item stand could display a holographic arrow when editing a handheld gun to clearer indicate the forward direction. It doesn't need to be very big or permanently visible, but something that slowly pulses perhaps would catch a new users eye
![image](https://user-images.githubusercontent.com/27857711/152577048-caa018d9-ae5a-4068-af90-54456b781aea.png)

**Comment by tsunamayo** [comment]
Yeah it is a very valid point. For weapon I still need to do the  / 2 reduction, and I think moving to a dedicated 3d software like (or like in many building game) and not a world based will make more sense.
